src/test/scala/uk/gov/hmrc/api/specs/CompleteMonthlyReturnsSpec.scala

```diff
@@ -121,33 +114,29 @@ class CompleteMonthlyReturnsSpec extends BaseSpec, LazyLogging {
     Given("I set the reporting windows as open and when no obligation has met")
     val isaReference = generateRandomZReference()
     disaReturnsStubService.setReportingWindow(true)
-    disaReturnsStubService.openObligationStatus(isaReference)
 
     Given("I created the client application and notification box")
     createClientApplication()
     createNotificationBoxAndSubscribe()
 
     When("I POST a request 'Initiate Returns Submission' API to get a returnId")
-    val taxYear          = "2025-26"
     val month            = "JAN"
     val initiateResponse = postInitiateReturnsSubmission(isaReference, month = month)
     initiateResponse.status shouldBe 200
     val initiateResponseJson = Json.parse(initiateResponse.body)
-    val returnId             = (initiateResponseJson \ "returnId").as[String]
 
     When("I POST a request 'Monthly Returns Submission' API")
     val monthlyReturnsSubmissionResponse =
-      postMonthlyReturnsSubmission(isaReference, returnId = returnId)
+      postMonthlyReturnsSubmission(isaReference)
 
     Then("I got the status code 204")
     monthlyReturnsSubmissionResponse.status shouldBe 204
 
-    When("I POST the 'Complete Monthly Returns' API")
+    When("I POST the 'Declaration request' API")
```

Comment:
I think this makes more sense as " I POST a request to the 'Declaration' endpoint"